src/main/java/hudson/tasks/junit/CaseResult.java

```diff
@@ -134,14 +134,24 @@ public CaseResult(SuiteResult parent, String testName, String errorStackTrace, S
     }
 
     @Restricted(Beta.class)
-    public CaseResult(SuiteResult parent, String className, String testName, String errorDetails, String skippedMessage, float duration) {
+    public CaseResult(
```

Comment:
ignoring compat break as it's beta and only user is sql plugin which will be released at the same time